### ISSUE TITLE
excalidraw+ can't change language

### ISSUE THREAD

**Issue by MangGoDio** [excalidraw/excalidraw#6353]
I've been using this tool for free for a while，I think this tool is so good that I bought excalidraw+ yesterday，But I found that you can't switch languages in excalidraw+，but you can switch languages in the free version，I am very strange.

**Comment by ad1992** [comment]
Probably this is a recent regression
cc @dwelle

**Comment by dwelle** [comment]
Yes, currently the Excalidraw+ editor is not multi-lingual as the site itself is not, but it doesn't hurt to have at least the editor translated, so we'll enable it.
